classes/modules/games_module.rb

```diff
@@ -0,0 +1,55 @@
+require_relative '../game'
+require_relative '../author'
+require_relative './author_module'
+require 'json'
+
+class GameModule
+  attr_accessor :games
+
+  def initialize
+    @storage_file = './data/games.json'
+    file = JSON.parse(File.read(@storage_file))
```

Comment:
- [ ] I can see you did solve the `No such file or directory` error for the `author_module.rb` file. However, `games_module.rb` file is still throwing an error is a file doesn't exist.

I will suggest you check this out. And try to solve the error for all modules that are making use of files.

See error screenshot |
:---------------------------:|
![a1](https://user-images.githubusercontent.com/39523312/165798840-4570e362-9203-4287-8d61-6d497b1b7ca3.png)